style.css

```diff
@@ -0,0 +1,1783 @@
+}
+
+.Container-2 {
+  display: flex;
+  padding: 0 20px;
+  background-color: #fff;
+  height: 615px;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.heading {
+  display: flex;
+  background-color: #fff;
+  grid-template-rows: auto;
+}
+
+.heading h1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 43px;
+  font-weight: 800;
+  font-style: normal;
+  line-height: 52px;
+  letter-spacing: 8px;
+  width: 100%;
+}
+
+.p-dis {
+  width: 100%;
+  background-color: #fff;
+  display: flex;
+}
+
+.p-dis p {
+  font-family: 'Poppins', sans-serif;
+  font-size: 21px;
+  font-weight: 400;
+  width: 100%;
+  justify-content: center;
+}
+
+/* .connect h3 {
+  color: #7f8cff;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+} */
+
+/* .social-media {
+  display: flex;
+  flex-wrap: 100px;
+  background-color: #fff;
+} */
+
+.social-media li {
+  list-style: none;
+  background-color: #fff;
+}
+
+.lang-1 img {
+  width: 8vh;
+}
+
+.language ul li {
+  border-radius: 8px;
+  background: #ebebff;
+  text-align: center;
+}
+
+.social-media li a img {
+  margin: 17px;
+}
+
+.lang-used {
+  width: 327px;
+  height: 320px;
+  display: flex;
+  justify-content: space-evenly;
+  flex-direction: column;
+  padding: 0 13px;
+}
+
+.space {
+  padding: 12px 0;
+}
+
+.navbar-2 {
+  list-style: none;
+  text-decoration: none;
+  background-color: #fff;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  color: black;
+  font-size: 30px;
+}
+
+.border {
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.left-2 {
+  display: flex;
+}
+
+.left-2 h3 {
+  margin-right: 12px;
+}
+
+.language-1 {
+  list-style: none;
+  display: grid;
+  gap: 20px;
+  grid-template-rows: auto auto auto;
+}
+
+.lang-1 h3 {
+  padding-left: 12px;
+  align-self: center;
+}
+
+.lang-used-img {
+  margin-left: 40%;
+}
+
+.Frame-1 {
+  border-bottom: 1px solid #dfe1e6;
+}
+
+/* The form starting here */
+.background-cont {
+  background: #6070ff;
+  background-image: url(./Pictures/contact\ form\ shapes-mobile@2x.png);
+  background-position: right;
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 779px;
+  width: 100%;
+  border-top-left-radius: 109px 91px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+.Frame-img {
+  margin: 2% 27%;
+}
+
+.skils-img {
+  margin: 2% 61%;
+}
+
+.Contact-me {
+  display: grid;
+  grid-template-rows: 75px 190px 74px 77px 153px 28px;
+  padding: 0 12px;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  color: white;
+}
+
+form legend {
+  font-size: 40px;
+  font-weight: 700;
+}
+
+.paraga-form {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  color: white;
+}
+
+.cont-input {
+  width: 327px;
+  height: 48px;
+  border: 1px solid #cfd8dc;
+  background-color: white;
+  border-radius: 8px;
+  font-size: 23px;
+  font-weight: 400;
+  padding: 8px;
+}
```

Comment:
- [ ] Please make sure that your design match the figma template its good for sharppening your skill and good for your future job.
### Your design👇
![contactst1111](https://user-images.githubusercontent.com/66992901/200015609-d6654e7d-0af4-440f-b112-06dc6616254b.PNG)
### figma design👇
![figmacontact](https://user-images.githubusercontent.com/66992901/200015681-16f968e9-0b80-40cb-8fc9-85aef4be5c68.PNG)